Luke.md

```diff
@@ -1,53 +1,59 @@
-        - aesigned the proposal: https://wiki.mozilla.org/Firefox_OS/Remote_Control#Secure_Connection
-        - arranged a con-call meetings with Paul Theriault
-        - futher discussion on IRC
-    - Confirmed and updated all the L10N strings with UX
+        - got the positive feedback from Paul Theriault
+        - finalized our proposal
+    - [Done] Bug 1212403 - [TV 2.5] Apply visual spec to the client-side page of TV remote control
+        - wrote the patch and got r+
+        - landed on master
+        - requesting the approval for v2.5
+    - [WIP] Bug 1212402 - [TV 2.5] Apply visual spec to the TV remote control app
+        - worte the patch and got r+
+        - visual designer would like to change the icon so suspended
+    - [WIP] Bug 1223387 - [TV 2.5][meta] Revise Remote Control string base on copy review result
+        - confirmed the final strings with UX and EPM
+    - [WIP] Bug 1197751 - [TV 2.5] Write a event shuttle in Gecko
+        - helped clarifying the idea of our proposal
 
 * [TV App Usage Metrics]
-    - Tested it on partner's real device but found some bugs. Have written a patch to workaround it
-    - Figured out the partner's native app working flow
-
-* [TV Audio Channel]
-    - the patch doesn't work on partner's branch
-    - dug into it with Kilik
+    - [New] Bug 1218327 - "installs" in payload of app usage metric is not recorded when installing packaged app from Marketplace
+        - helped on porting the patch to TV build
+        - couldn't pass QA's test
+        - handed over to Rex
 
 * [TV Marketplace Workflow]
-    - Helped Ricky on AppWindowManager and Key Event issues
-    - Answered questions via mail
+    - Attended the meetings due to partner's request
+    - Keep helping Ricky on implementing this feature
+
+* [TV System]
+    - Attended the meetings Related to partner's porting issues
 
 * [Firefox Developers Conference 2015]
-    - Refined the SpatialNavigation library
+    - https://github.com/luke-chang/js-spatial-navigation
+    - Removed the dependency of jQuery from the library and Refactored the library
     - Prepared slides
+    - Presented a Brown bag
 
 ### This Week ###
 
+* [Firefox Developers Conference 2015]
+    - Give a presentation about TV Optimized Web App
+    - http://www.mozilla.jp/events/devcon/2015/tokyo/
+
 * [TV Remote Control]
-    - [WIP] Bug 1215457 - [TV 2.5] Secure connection between remote control server and client
+    - [WIP] Bug 1212402 - [TV 2.5] Apply visual spec to the TV remote control app
 
-* [TV App Usage Metrics]
-    - follow up porting issues
+* [PTO]
+    - 11/17 - 11/19
 
-* [TV Audio Channel]
-    - follow up porting issues
+## 2015 Q4 Goals ##
 
-* [Firefox Developers Conference 2015]
-    - Refined the SpatialNavigation library
-    - Prepared slides
+1. TV Remote Control implementation, as measured by implementation completeness. 60%
+2. TV partner support, as measured by number of issues handled. 10%
```

Comment:
We won't measure a goal by bug counts - it doesn't make sense. I will say this item isn't measurable, or we could define it in another way.